formhero/forms/serializers.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+from formhero.forms.models import Form
+
+class MessageSerializer(serializers.ModelSerializer):
+
+
+    class Meta:
+        model = Form
+        fields = (
+
+        )
```

Comment:
This looks like it's been taken from commute crush? Make sure and be careful when copying code. I don't think this file is even needed.